misc/python/materialize/postgres_consistency/ignore_filter/pg_inconsistency_ignore_filter.py

```diff
@@ -180,6 +180,9 @@ def _matches_problematic_function_invocation(
         ):
             return YesIgnore("#25937 (base64 decode with new line and tab)")
 
+        if db_function.function_name_in_lower_case == "translate":
```

Comment:
I would like to make it more fine-grained that to exclude the whole function unconditionally. Let me have a deeper look at the issue and I will come up with a suggestion.

Reply:
Updated the PR accordingly.

I limited it to the second parameter; if this turns out not to be enough, we can change it to `expression.has_any_characteristic` to apply it to all parameters.